ios/browser/api/ads/brave_ads.mm

```diff
@@ -64,24 +64,23 @@ +(void)__objc_setter : (__type)newValue {                                \
 
 static NSString* const kLegacyAdsEnabledPrefKey = @"BATAdsEnabled";
 static NSString* const kLegacyNumberOfAdsPerHourKey = @"BATNumberOfAdsPerHour";
-static NSString* const kLegacyShouldAllowAdsSubdivisionTargetingPrefKey =
+static NSString* const kLegacyShouldAllowSubdivisionTargetingPrefKey =
     @"BATShouldAllowAdsSubdivisionTargetingPrefKey";
-static NSString* const kLegacyAdsSubdivisionTargetingCodePrefKey =
+static NSString* const kLegacySubdivisionTargetingCodePrefKey =
     @"BATAdsSubdivisionTargetingCodePrefKey";
-static NSString* const kLegacyAutoDetectedAdsSubdivisionTargetingCodePrefKey =
+static NSString* const kLegacyAutoDetectedSubdivisionTargetingCodePrefKey =
     @"BATAutoDetectedAdsSubdivisionTargetingCodePrefKey";
 
 static NSString* const kAdsEnabledPrefKey =
     base::SysUTF8ToNSString(ads::prefs::kEnabled);
 static NSString* const kNumberOfAdsPerHourKey =
     base::SysUTF8ToNSString(ads::prefs::kAdsPerHour);
-static NSString* const kShouldAllowAdsSubdivisionTargetingPrefKey = [NSString
-    stringWithUTF8String:ads::prefs::kShouldAllowAdsSubdivisionTargeting];
-static NSString* const kAdsSubdivisionTargetingCodePrefKey =
-    base::SysUTF8ToNSString(ads::prefs::kAdsSubdivisionTargetingCode);
-static NSString* const kAutoDetectedAdsSubdivisionTargetingCodePrefKey =
-    [NSString stringWithUTF8String:
-                  ads::prefs::kAutoDetectedAdsSubdivisionTargetingCode];
+static NSString* const kShouldAllowSubdivisionTargetingPrefKey = [NSString
+    stringWithUTF8String:ads::prefs::kShouldAllowSubdivisionTargeting];
```

Comment:
`base::SysUTF8ToNSString` instead of `[NSString stringWithUTF8String:]`

Reply:
Fixed